.eslintrc.js

```diff
@@ -23,5 +23,6 @@ module.exports = {
     'react',
   ],
   rules: {
+    "react/no-array-index-key": 0
```

Comment:
I have several simple map functions, for example where I have a section of 'Troops' and if they user wants 2 Necron warrior squads, then I can't use unique ID's as key's because they have the same key.  So I used unique ID's as keys when I could, but instances like this where I don't have or can't use unique ID's in the map function, that is why I had that rule ignored.  What do you suggest?

Reply:
Hmmm, what about making each squad a unique ID from the database? There is also this great little utility I use for generating unique IDs locally https://www.npmjs.com/package/uuid